addons/sale_timesheet/static/src/components/so_line_field/so_line_field.js

```diff
@@ -11,7 +11,10 @@ export class SoLineField extends Many2OneField {
         const update = this.update;
         this.update = (value, params = {}) => {
             update(value, params);
-            if (value || this.updateOnEmpty) {
+            if ( // field is unset AND the old & new so_lines are different
+                !this.props.record.data.is_so_line_edited &&
```

Comment:
I guess it does or this wouldn't be done, but does it matter if we do the update if it's already the current value

Reply:
My guess is that it pre-empts an unnecessary RPC, but it's my first JS ticket, so not sure why it's done elsewhere either.